docs/api.md

```diff
@@ -766,12 +766,23 @@ var argv = require('yargs')
 
 .example(cmd, desc)
 -------------------
+.example(exampleArray)
```

Comment:
```suggestion
.example([[cmd1, desc1], [cmd2, desc2]])
```

☝️  what do you think?

Reply:
```suggestion

.example([[cmd1, desc1], [cmd2, desc2], ...])
```

Maybe that (added 3 dots to let them know it can be any size)?